gradle.properties

```diff
@@ -21,4 +21,4 @@ publishRepo=
 bintrayOrgName=pravega
 bintrayRepoName=pravega
 
-buildVersion=0.6.0-SNAPSHOT
+buildVersion=0.6.1-SNAPSHOT
```

Comment:
Should not that say 0.7.0-SNAPSHOT?

Reply:
Thanks @sarlaccpit. Yes, it should say `0.7.0-SNAPSHOT`. I've pushed a commit with this change.